### PR TITLE
Most flake8 warnings corrected

### PR DESCRIPTION
**Pull request by oddstr13**
TODO
---
- [x] Correct SQL indentation
- [x] Investigate unused imports and variables

### CODE REVIEW

**Comment by oddstr13** [comment]
Didn't feel like fixing all the SQL indentation by hand, so [I made a script][1]
`python3 ../jf-kodi-sqlite-statement-indent.py $(find . -name 'queries*.py')`

[1]: https://gist.github.com/oddstr13/9265590f1591f2ef750e4d2bedd35eb1

**Comment by oddstr13** [comment]
Still some `F841 local variable is assigned to but never used` left, but fixing those probably requires more knowledge of the code in question than I care for right now.

The following deserve their own issue and a proper fix;
```
./resources/lib/full_sync.py:514:35: F821 undefined name 'Movies'
./resources/lib/full_sync.py:522:35: F821 undefined name 'TVShows'
```
Looks like it is related to mixed content libraries.
@mcarlton00 & @TrueTechy you guys touched this recently, and probably have a better understanding than me of this particular part of the code

File: resources/lib/library.py [hunk 1]
```diff
@@ -159,20 +156,19 @@ def service(self):
...
                     self.progress_updates = xbmcgui.DialogProgressBG()
                     self.progress_updates.create(_('addon_name'), _(33178))
-                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates))*100), message="%s: %s" % (_(33178), queue_size))
+                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates)) * 100), message="%s: %s" % (_(33178), queue_size))
                 elif queue_size:
-                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates))*100), message="%s: %s" % (_(33178), queue_size))
+                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates)) * 100), message="%s: %s" % (_(33178), queue_size))
                 else:
-                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates))*100), message=_(33178))
+                    self.progress_updates.update(int((float(self.total_updates - queue_size) / float(self.total_updates)) * 100), message=_(33178))
```
Comment (mcarlton00) [inline resources/lib/library.py]: More longer lines.  Again, not super strong opinions, my editor just highlights them and it's slightly annoying.  We probably should figure out a consensus on coding standards at some point as more people start working on this.

File: resources/lib/objects/music.py [hunk 2]
```diff
@@ -269,15 +258,13 @@ def song(self, item, e_item):
...
-
         if update:
             self.song_update(obj)
         else:
             self.song_add(obj)
 
-
         self.link_song_album(*values(obj, QU.update_song_album_obj))
-        self.add_role(*values(obj, QU.update_role_obj)) # defaultt role
+        self.add_role(*values(obj, QU.update_role_obj))  # defaultt role
```
Comment (mcarlton00) [inline resources/lib/objects/music.py]: Nitpicky spelling error that I'm betting we inherited

File: resources/lib/database/__init__.py [hunk 3]
```diff
@@ -105,8 +105,7 @@ def _discover_database(self, database):
 
         for file in reversed(files):
 
-            if (file.startswith(database) and not file.endswith('-wal') and
-                not file.endswith('-shm') and not file.endswith('db-journal')):
+            if (file.startswith(database) and not file.endswith('-wal') and not file.endswith('-shm') and not file.endswith('db-journal')):
```
[the remark this answers is not in the document]
Comment (oddstr13) [reply resources/lib/database/__init__.py]: flake8 was complaining about line break around the operators before this change, so..

I specifically disabled line length (9999) when I first set up flake8

File: resources/lib/objects/music.py (hunk 2 above)
Comment (oddstr13) [reply resources/lib/objects/music.py]: In my defence, I was in full flake8 syntax mode when speeding past this line, all human text processing was disabled at the time :stuck_out_tongue_winking_eye:

File: resources/lib/library.py (hunk 1 above)
Comment (oddstr13) [reply resources/lib/library.py]: I'm not a huge fan of arbitrary line length limits
editorconfig `max_line_length = off` may do the trick? It's set to `null` right now.

**Pull request merged**